src/components/shared/ProfileCallTreeContextMenu.js

```diff
@@ -395,10 +418,14 @@ class ProfileCallTreeContextMenu extends PureComponent<Props> {
           Drop samples with this function
         </MenuItem>
         <div className="react-contextmenu-separator" />
-        <MenuItem onClick={this.handleClick} data={{ type: 'expand-all' }}>
-          Expand all
-        </MenuItem>
-        <div className="react-contextmenu-separator" />
+        {showExpandAll ? (
+          <div>
+            <MenuItem onClick={this.handleClick} data={{ type: 'expand-all' }}>
+              Expand all
+            </MenuItem>
+            <div className="react-contextmenu-separator" />
+          </div>
```

Comment:
I hope this `div` is okay to add.

Reply:
You could add a React `Fragment` here. They are new to React 16, which we recently upgraded to.

https://reactjs.org/docs/fragments.html